app/Repositories/IndependentActivity/IndependentActivityRepository.php

```diff
@@ -1024,7 +1025,7 @@ public function moveToPlaylist($independentActivity, $playlist, $token)
             'order' => $this->getOrder($playlist->id) + 1,
             'h5p_content_id' => $independentActivity->h5p_content_id, // Move the content 
             'thumb_url' => $newThumbUrl,
-            'shared' => 0,
+            'shared' => Project::where('id', $playlist->project_id)->value('shared'),
```

Comment:
The "shared" for the project can be get using following code

$playlist->project->shared